src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.UnoPlatform/Sandbox.UnoPlatform/Properties/launchSettings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8080",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    // This profile is first in order for dotnet run to pick it up by default
+    "Sandbox.Unplatform (WebAssembly)": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:5000",
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "Sandbox.Unplatform (WebAssembly IIS Express)": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    // Note: In order to select this profile, you'll need to comment the `Packaged` profile below until this is fixed: https://aka.platform.uno/wasdk-maui-debug-profile-issue
+    "Sandbox.Unplatform (WinAppSDK Unpackaged)": {
+      "commandName": "Project",
+      "compatibleTargetFramework": "windows"
+    },
+    "Sandbox.Unplatform (WinAppSDK Packaged)": {
+      "commandName": "MsixPackage",
+      "compatibleTargetFramework": "windows"
+    },
+    "Sandbox.Unplatform (Desktop)": {
+      "commandName": "Project",
+      "compatibleTargetFramework": "desktop"
+    },
+    "Sandbox.Unplatform (Desktop WSL2)": {
```

Comment:
I have tested all platforms. However, I am currently unable to launch **Desktop WSL2 with the Sandbox** due to a space in the path. 

For related details on the Uno Platform side, please refer to:
- This comment: [unoplatform/private#479 (comment).](https://github.com/unoplatform/private/issues/479#issuecomment-2221864398)).
- The current documentation: https://platform.uno/docs/articles/common-issues-vs2022.html#my-application-does-not-start-under-wsl

I am uncertain about the best approach to address this issue. Here are a few options:

1. Change the folder name to remove the space and fix the issue.
2. Leave a comment in MainPage.xaml or another relevant file explaining this issue.
3. Do you have any other suggestions please?

![image](https://github.com/user-attachments/assets/3cce089c-721f-453c-9bf4-5e962833d05b)

Thanks in advance for the feedback

Reply:
https://github.com/ScottPlot/ScottPlot/pull/4123#issuecomment-2311250146